### ISSUE TITLE
Missing HTML-Attributes

### ISSUE THREAD

**Issue by Suven** [softlayer/sl-ember-components#680]
Hey,

I noticed that some attributes, which would be possible via plain HTML are currently not supported by many components.

For example the input-component is missing support for tabindex and pattern.

Maybe it makes sense to have a bulk-pr for this. What do you think?

**Comment by notmessenger** [comment]
I agree with this request.  I have added this issue to the https://github.com/softlayer/sl-ember-components/milestones/v0.14.0%20(User%20Experience) milestone

**Comment by notmessenger** [comment]
https://github.com/softlayer/sl-ember-components/pull/688 is the PR that was submitted for this work.
